derive/src/lib.rs

```diff
@@ -23,7 +32,43 @@ macro_rules! ident {
     };
 }
 
-fn expand_contract(input: LitStr) -> Result<TokenStream> {
+struct ContractArgs {
+    artifact_path: LitStr,
+    runtime_crate: Option<Ident>,
+}
+
+impl Parse for ContractArgs {
+    fn parse(input: ParseStream) -> ParseResult<Self> {
+        let mut result = ContractArgs {
+            artifact_path: input.parse()?,
+            runtime_crate: None,
+        };
+
+        while !input.is_empty() {
+            input.parse::<Token![,]>()?;
+            if input.is_empty() {
+                break;
+            }
+
+            let param = input.call(Ident::parse_any)?;
+            input.parse::<Token![=]>()?;
```

Comment:
What's the difference between `Token![=]` and `Token![,]`?

Reply:
`Token![=]` is the `=` token and `Token![,]` is the `,` token.